www/grpc/proto/zarb.proto

```diff
@@ -126,4 +120,51 @@ message Peer{
   int32 received_messages = 7;
   int32 invalid_messages = 8;
   int32 received_bytes = 9;
+}
+
+message Account{
+  string address = 1;
+  int32 number = 2;
+  int32 sequence = 3;
+  int64 Balance = 4;
+}
+
+message BlockInfo {
+  int32 version = 1;
+  string last_block_hash = 2;
+  string state_hash=3;          
+  string tx_ids_hash=4;
+  string last_certificate_hash=5;
+  bytes sortition_seed=6;
+  string proposer_address=7;   
+  int64 round=8;  
+  repeated int32 committers=9;
+  repeated int32 absentees=10;
+  string signature=11;
+}
+
+message Transaction{
+  string id=1;    
+  int32 Version =2;  
+  string stamp=3;    
+  int64 sequence=4; 
```

Comment:
Since sequence in account & validator are almost same as sequence in transaction, type also should be the same. 
here type in `int64` but in account & validator is `int32`.